localstack/services/firehose/firehose_api.py

```diff
@@ -61,7 +61,10 @@ def put_records(stream_name, records):
         if 'ESDestinationDescription' in dest:
             es_dest = dest['ESDestinationDescription']
             es_index = es_dest['IndexName']
-            es_type = es_dest['TypeName']
+            try:
+                es_type = es_dest['TypeName']
```

Comment:
nitpick: Can we use this instead?
```
es_type = es_dest.get('TypeName')
```

Reply:
Hey, thanks for the suggestion. I have updated it.